CuitService.Test/TaxInfoApiTest.cs

```diff
@@ -145,8 +145,10 @@ public async Task GET_taxinfo_by_cuit_should_make_honor_original_format()
             var content = await response.Content.ReadAsStringAsync();
 
             // Assert
-            // Validate indentation
+            // Validate json format
             Assert.Matches(@"(?<=\"")([^\s,].*?)(?=\"")|null", content);
+            // Validate indentation
+            Assert.Matches(@"^{\n\s+""(.+\n)*}$", content);
```

Comment:
Do you have any idea that in my local environment this assertion fails?
![image](https://user-images.githubusercontent.com/6796523/114396500-078b1880-9b74-11eb-974a-3caa2140231a.png)

Reply:
Tal vez que tengas salto de línea de windows?

¿Querés probar esto?

```suggestion
            Assert.Matches(@"^{\r?\n\s+""(.+\r?\n)*}$", content);
```

Reply:
Joya era eso, no lo podía hacer andar este test gracias

Reply:
Lo acomodo y mergeo entonces, gracias!